sw/device/lib/dif/dif_plic.c

```diff
@@ -201,7 +201,7 @@ static size_t plic_num_priority_reg(void) { return kDifPlicIrqIdLast; }
  */
 static ptrdiff_t plic_priority_reg_offset(dif_plic_irq_id_t irq) {
   ptrdiff_t offset = PLIC_ID_TO_INDEX(irq) * sizeof(uint32_t);
-  return RV_PLIC_PRIO0_REG_OFFSET + offset;
+  return RV_PLIC_PRIO1_REG_OFFSET + offset;
```

Comment:
Revised this

Reply:
I don't think this solves the fact that some of the offsets are calculated from 1 and some from 0, but I also don't think we're so far from fully solving that.

I think if we now also define a `PRIO0` register (before `PRIO1` in memory) that does not cause effects in the plic, but can be read from and written to without erroring, then the `offset` calculations no longer have the off-by-one issue. Does this make sense?